src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -66,17 +66,20 @@ public function generateSiteConfigFiles() {
       $project_local_drush_file = "$multisite_dir/local.drushrc.php";
 
       $copy_map = [
-        $default_project_default_settings_file => $project_default_settings_file,
         $blt_local_settings_file => $default_local_settings_file,
         $default_local_settings_file => $project_local_settings_file,
         $blt_local_drush_file => $default_local_drush_file,
         $default_local_drush_file => $project_local_drush_file,
       ];
 
       // Only add the settings file if the default exists.
-      if (file_exists($project_default_settings_file)) {
+      if (file_exists($default_project_default_settings_file)) {
+        $copy_map[$default_project_default_settings_file] = $project_default_settings_file;
         $copy_map[$project_default_settings_file] = $project_settings_file;
       }
+      else {
+        $this->yell("No $default_project_default_settings_file file found.");
```

Comment:
I added this "yell" just to make sure the user is aware that an expected file is missing.